vendor/github.com/ciscoecosystem/mso-go-client/models/schema_template_vrf.go

```diff
@@ -6,15 +6,17 @@ type SchemaTemplateVrf struct {
 	Value map[string]interface{} `json:",omitempty"`
 }
 
-func NewSchemaTemplateVrf(ops, path, Name, displayName string, l3m, vzany bool) *SchemaSite {
+func NewSchemaTemplateVrf(ops, path, Name, displayName, ipDataPlaneLearning string, l3m, vzany, preferredGroup bool) *SchemaSite {
 	var VrfMap map[string]interface{}
 
 	if ops != "remove" {
```

Comment:
Is this condition still required?

Reply:
yes.

Reply:
Since we're using -> vrfRemovePatchPayload := models.GetRemovePatchPayload("/templates/" + template + "/vrfs/" + name) in the provider

we can remove the if and else condition as they are no longer needed.
If it's a hassle let it be